.gitignore

```diff
@@ -1,15 +1,16 @@
 *.idea/
 .tox/
+manager.cfg
 Application_Manager.egg-info/
 dist/
 build/
 *.pyc
-manager.cfg
 manager.cfg.lsd
-*.un~
+*.un
```

Comment:
why change this?

Reply:
I think this change should be ignored.

Reply:
I search about the ".un" extension and the right form is ".un~". It's a file created by vim.